src/themes/elife/styles.css

```diff
@@ -158,6 +158,22 @@ main {
   text-transform: capitalize;
 }
 
+:--references {
```

Comment:
@discodavey since this was written the DOM has been corrected so the downloads are not within `:--references` any more. As you're busy with the other thing I'll pick up this PR and finish it off.